specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5214 @@
+            "modelAsString": false
+          }
+        },
+        "createdTime": {
+          "format": "date-time",
+          "description": "The time the service was created.",
+          "type": "string"
+        },
+        "updatedTime": {
+          "format": "date-time",
+          "description": "The time the service was updated.",
+          "type": "string"
+        },
+        "error": {
+          "description": "The error details.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          ]
+        },
+        "computeType": {
+          "description": "The compute environment type for the service.",
+          "enum": [
+            "ACI",
+            "AKS"
+          ],
+          "type": "string",
+          "example": "AKS",
+          "x-ms-enum": {
+            "name": "ComputeEnvironmentType",
+            "modelAsString": true
+          }
+        },
+        "deploymentType": {
+          "description": "The deployment type for the service.",
+          "enum": [
+            "GRPCRealtimeEndpoint",
+            "HttpRealtimeEndpoint",
+            "Batch"
+          ],
+          "type": "string",
+          "example": "HttpRealtimeEndpoint",
+          "x-ms-enum": {
+            "name": "DeploymentType",
+            "modelAsString": false
+          }
+        }
+      },
+      "discriminator": "computeType"
+    },
+    "PaginatedServiceList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning service objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "description": "An array of Machine Learning compute objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/ServiceResource"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "ACIServiceResponse": {
+      "description": "The response for an ACI service.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ServiceResponseBase"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "containerResourceRequirements": {
+              "description": "The container resource requirements.",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/ContainerResourceRequirements"
+                }
+              ]
+            },
+            "scoringUri": {
+              "description": "The Uri for sending scoring requests.",
+              "type": "string"
+            },
+            "location": {
+              "description": "The location of the service.",
+              "type": "string"
+            },
+            "authEnabled": {
```

Comment:
please see all the linter errors related to the new booleans.  Enums with "Enabled"/"Disabled" options are preferred